### DISCUSSION TITLE
Glog upgrade breaks folly

### DISCUSSION THREAD

**Discussion by bobfang1992** [xmake-io/xmake-repo#4221] [General]
Hi,

For folly it depends on glog:
https://github.com/xmake-io/xmake-repo/blob/a5e98dbb129971e21eef7e451846026690367c10/packages/f/folly/xmake.lua#L35
But recently there is a new version of glog added:
https://github.com/xmake-io/xmake-repo/blame/a5e98dbb129971e21eef7e451846026690367c10/packages/g/glog/xmake.lua#L9

And this version is no longer compatible with folly.

I am wondering how I can fix this issue? I want force folly to use 0.6.0 if possible.

Thanks!

**Comment by waruqi** [comment]
https://github.com/xmake-io/xmake-repo/pull/4228

**Comment by waruqi** [comment]
try it again.
